File: C++/basic.cpp
#include <iostream>
#include "basic.h"
using namespace std;

// add() function implementation - all function declarations located in basic.h
int basic::add(float num1, float num2) {
    float num3;
    num3 = num1 + num2;
    cout << num3;
    return 0;
}

// subtract() function implementation
int basic::subtract(float num1, float num2) {
    float num3;
    num3 = num1 - num2;
    cout << num3;
    return 0;
}

// multiply() function implementation
int basic::multiply(float num1, float num2) {
    float num3;
    num3 = num1 * num2;
    cout << num3;
    return 0;
}

// divide() function implementation
int basic::divide(float num1, float num2) {
    float num3;
    num3 = num1 / num2;
    cout << num3;
    return 0;
}

File: C#/Basic.cs
using static System.Convert;
using static System.Console;

namespace CalculatorCS
{
    public class Basic
    {
        public static void Add(int num1, int num2)
        {
            int num3 = num1 + num2;
            WriteLine("Answer: " + num3);
        }

        public static void Subtract(int num1, int num2)
        {
            int num3 = num1 - num2;
            WriteLine("Answer: " + num3);  
        }

        public static void Multiply(int num1, int num2)
        {
            int num3 = num1 * num2;
            WriteLine("Answer: " + num3);
        }

        public static void Divide(int num1, int num2)
        {
            int num3 = num1 / num2;
            WriteLine("Answer: " + num3);
        }
    }
}